lib/mock.js

```diff
@@ -86,12 +90,15 @@ exports.deleteObjects = function (search, callback) {
 
 	var deleted = [];
 	var errors = [];
+	var basePath = exports._options.basePath;
```

Comment:
Why create a variable for the `basePath` when it can be accessed from the options object? Just to clean up the `path.join` below? Not sure it's necessary.
(this is relevant to all other instances of that variable)